controllers/loginController.js

```diff
@@ -1,13 +1,15 @@
 const crypto = require('crypto');
 const { check } = require('express-validator');
 const validation = require('../utils/validation')
+const demarchesSimplifiees = require('../utils/demarchesSimplifiees')
 const dbPsychologists = require('../db/psychologists');
 const dbLoginToken = require('../db/loginToken');
 const date = require('../utils/date');
 const cookie = require('../utils/cookie');
 const emailUtils = require('../utils/email');
 const ejs = require('ejs');
 const config = require('../utils/config');
+const { demarchesSimplifieesId } = require('../utils/config');
```

Comment:
'demarchesSimplifieesId' is assigned a value but never used

Reply:
`const { demarchesSimplifieesId } = require('../utils/config');` :see_no_evil: